tests/everest/test_everest_client.py

```diff
@@ -190,10 +190,14 @@ def everserver_is_running():
         # Connect to the websockets endpoint
         client_event_queue, monitor_thread = client.setup_event_queue_from_ws_endpoint()
         client_event_queues.append(client_event_queue)
+        monitor_threads.append(monitor_thread)
         monitor_thread.start()
 
     # Wait until the server has finished running the simulation
     everest_main_thread.join()
+    for _thread in monitor_threads:
+        if _thread.is_alive():
+            _thread.join(timeout=5)
 
```

Comment:
I guess the thread can still be alive here if the timeout has expired but maybe this is not important to check

Reply:
Not really important if the thread is still alive after the timeout, no exception will be raised this was added just to make sure that monitor threads have enough time to process the events before we check that all events are received.